x-pack/test/plugin_api_integration/plugins/sample_task_plugin/server/plugin.ts

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Plugin, CoreSetup, CoreStart } from 'kibana/server';
+import { EventEmitter } from 'events';
+import { Subject } from 'rxjs';
+import { first } from 'rxjs/operators';
+import { initRoutes } from './init_routes';
+import {
+  TaskManagerSetupContract,
+  TaskManagerStartContract,
+  ConcreteTaskInstance,
+} from '../../../../../plugins/task_manager/server';
+import { DEFAULT_MAX_WORKERS } from '../../../../../plugins/task_manager/server/config';
+
+// this plugin's dependendencies
+export interface SampleTaskManagerFixtureSetupDeps {
+  taskManager: TaskManagerSetupContract;
+}
+export interface SampleTaskManagerFixtureStartDeps {
+  taskManager: TaskManagerStartContract;
+}
+
+export class SampleTaskManagerFixturePlugin
```

Comment:
This is just the legacy plugin copy-pasted into a NP plugin.
I couldn't convince Git that this is the same file though it is 95% the same code...